framework/Models/PostProcessors/PostProcessor.py

```diff
@@ -109,21 +110,27 @@ def __init__(self, runInfoDict):
     self.action = None            # action
     self.workingDir = ''          # path for working directory
     self.printTag = 'POSTPROCESSOR MODEL'
-
-  # use BaseClass method provideExpectedMetaKeys and addMetaKeys to provide and modify metadata
+    self.outputDataset  = False # True if the user wants to dump the outputs to dataset
+    self.validDataType = ['PointSet','HistorySet'] # The list of accepted types of DataObject
```

Comment:
what about dataset?

Reply:
In most of our PP, we only allow PointSet and HistorySet. In each specific PP, we can specific the accepted types.